packages/core/src/components/TermList/Term.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+import Row from '../Row';
+import Spacing from '../Spacing';
+import Text from '../Text';
+
+export default function Term({
+  label,
+  after = '',
+  endAlign,
+  regular,
+  children,
+}: {
+  label: string;
+  after?: React.ReactNode;
+  endAlign?: boolean;
+  regular?: boolean;
+  children: NonNullable<React.ReactNode>;
```

Comment:
one thing that both of these new components are missing is the props being declared above, not inline with the params and the descriptions for each prop are missing, List for example:

```jsx
export type Props = {
  /** List items. */
  children: NonNullable<React.ReactNode>;
  /** Apply gutters between `li`s. */
  gutter?: boolean;
  /** Horizontal list. */
  horizontal?: boolean;
  /** Align contents in the middle vertically, to be used with `horizontal`. */
  middleAlign?: boolean;
  /** Renders an `<ol></ol>`. */
  ordered?: boolean;
  /** Wrap horizontal list. */
  wrap?: boolean;
};

export default function List({ children, gutter, horizontal, middleAlign, ordered, wrap }: Props) {
```